kinder/pkg/cluster/status/cluster.go

```diff
@@ -119,6 +141,21 @@ func FromDocker(name string) (c *Cluster, err error) {
 	return c, nil
 }
 
+// Name returns the cluster's name
+func (c *Cluster) Name() string {
+	return c.name
+}
+
+// KubeConfigPath returns the path to where the Kubeconfig would be placed
+// by kinder based on the configuration.
+func (c *Cluster) KubeConfigPath() string {
+	// configDir matches the standard directory expected by kubectl etc
+	configDir := filepath.Join(homedir.HomeDir(), ".kube")
+	// note that the file name match kind config files. Might be in future we want to change this
```

Comment:
"Maybe we want to change this in the future."

Reply:
done